### DISCUSSION TITLE
Hide menu option for subtitles

### DISCUSSION THREAD

**Discussion by zerobertel** [TheWidlarzGroup/react-native-video#4957] [Ideas]
### What problem are you trying to solve?

When there are no subtitles available for a video, the subtitles menu option is still shown but has no functionality. The option appears but does nothing.

### Proposed solution

Add a "hide" param for the subtitles menu option, so the option can be hidden when no subtitle is available.

### Platforms

iOS

### Additional context

Reported by Robert.

Screenshot: ![Simulator Screen Shot - iPhone 14 - 2023-04-20 at 16 06 11](https://user-images.githubusercontent.com/10117109/233392462-778a5150-3cc7-4892-a5ff-05c8fdbcc92f.png)

**Comment by freeboub** [comment]
Not sure there is an easy way to achieve that. (I had a quick look in AVPlayer doc)
To be clear:
- Do the video contains subtitle tracks ?
- Is it limited to ios ?
Thank you

**Comment by TWG-AI** [comment]
Thanks for the idea — noted for the team to consider.

> I'm an AI assistant, so double-check any specifics I may have mentioned.
